sync.go

```diff
@@ -314,6 +344,23 @@ func LastLit(recs protocol.Records) byte {
 	return protocol.Lit(recs[len(recs)-1])
 }
 
+func (sync *Syncer) resetPingTimer() {
+	sync.lock.Lock()
+	defer sync.lock.Unlock()
+	if sync.pingTimer != nil {
+		sync.pingTimer.Stop()
+	}
+	sync.pingTimer = time.AfterFunc(sync.PingPeriod, func() {
+		sync.lock.Lock()
+		if sync.feedState == SendDiff || sync.feedState == SendLive {
+			sync.lock.Unlock()
+			sync.SetFeedState(SendPing)
```

Comment:
boolean flag is a good idea. the second concern is not a problem because if feed is completely blocked, then this replica will miss the ping from the other side and the other side will break the connection